routing_model/src/flows_dst.rs

```diff
@@ -346,3 +346,58 @@ impl CheckAndProcessElderChange {
         self.clone()
     }
 }
+
+#[derive(Debug, PartialEq, Clone)]
+pub struct CheckOnlineOffline(pub State);
+
+impl CheckOnlineOffline {
```

Comment:
Maybe indicate the commit the reference diagram was at in your PR
I assume: https://raw.githack.com/maidsafe/routing/bd2d7d32cb1fcffc57e652df3fa5ff0f92f430fe/routing_model/index.html